.circleci/docker/build_docker.sh

```diff
@@ -49,6 +50,13 @@ fi
 # Only push if `DOCKER_SKIP_PUSH` = false
 if [ "${DOCKER_SKIP_PUSH:-true}" = "false" ]; then
   docker push "${image}:${tag}"
+
+  if [ "${PUSH_GHCR_IMAGE:-}" ]; then
+    # Push docker image to the ghcr.io
+    echo $GHCR_PAT | docker login ghcr.io -u pytorch --password-stdin
```

Comment:
GHCR_PAT would be blank on forked pull requests, we might need to pass it from the top level workflow